frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -93,7 +177,7 @@ export const surveysLogic = kea<surveysLogicType>([
         ],
     }),
     afterMount(async ({ actions }) => {
-        await actions.loadSurveys()
-        await actions.loadResponsesCount()
```

Comment:
This can (and should) happen concurrently

Reply:
let's remove the async right above too then~